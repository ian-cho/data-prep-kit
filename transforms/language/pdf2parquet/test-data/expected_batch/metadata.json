{
  "pipeline": "pipeline_id",
  "job details": {
    "job category": "preprocessing",
    "job name": "pdf2parquet",
    "job type": "pure python",
    "job id": "job_id",
    "start_time": "2025-02-10 14:45:21",
    "end_time": "2025-02-10 14:45:28",
    "status": "success"
  },
  "code": {
    "github": "github",
    "commit_hash": "12345",
    "path": "path"
  },
  "job_input_params": {
    "batch_size": 10,
    "artifacts_path": null,
    "contents_type": "text/markdown",
    "do_table_structure": true,
    "do_ocr": true,
    "ocr_engine": "easyocr",
    "bitmap_area_threshold": 0.05,
    "pdf_backend": "dlparse_v2",
    "double_precision": 0,
    "checkpointing": false,
    "max_files": -1,
    "random_samples": -1,
    "files_to_use": [
      ".pdf",
      ".docx",
      ".pptx",
      ".zip"
    ],
    "num_processors": 0
  },
  "execution_stats": {
    "cpus": 28.6,
    "gpus": 0,
    "memory": 24.32,
    "object_store": 0,
    "execution time, min": 0.113
  },
  "job_output_stats": {
    "source_files": 2,
    "source_size": 605137,
    "result_files": 1,
    "processing_time": 3.426,
    "nrows": 3,
    "nsuccess": 3,
    "nfail": 0,
    "nskip": 0,
    "result_size": 26903
  },
  "source": {
    "name": "/Users/dol/codes/data-prep-kit/transforms/language/pdf2parquet/test-data/input",
    "type": "path"
  },
  "target": {
    "name": "/Users/dol/codes/data-prep-kit/transforms/language/pdf2parquet/output",
    "type": "path"
  }
}