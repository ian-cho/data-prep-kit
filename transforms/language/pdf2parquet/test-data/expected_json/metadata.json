{
  "pipeline": "pipeline_id",
  "job details": {
    "job category": "preprocessing",
    "job name": "pdf2parquet",
    "job type": "pure python",
    "job id": "job_id",
    "start_time": "2025-02-10 14:44:43",
    "end_time": "2025-02-10 14:44:50",
    "status": "success"
  },
  "code": {
    "github": "github",
    "commit_hash": "12345",
    "path": "path"
  },
  "job_input_params": {
    "batch_size": -1,
    "artifacts_path": null,
    "contents_type": "application/json",
    "do_table_structure": true,
    "do_ocr": true,
    "ocr_engine": "easyocr",
    "bitmap_area_threshold": 0.05,
    "pdf_backend": "dlparse_v2",
    "double_precision": 0,
    "checkpointing": false,
    "max_files": -1,
    "random_samples": -1,
    "files_to_use": [
      ".pdf",
      ".docx",
      ".pptx",
      ".zip"
    ],
    "num_processors": 0
  },
  "execution_stats": {
    "cpus": 28.5,
    "gpus": 0,
    "memory": 24.53,
    "object_store": 0,
    "execution time, min": 0.107
  },
  "job_output_stats": {
    "source_files": 2,
    "source_size": 605137,
    "result_files": 2,
    "result_size": 23484,
    "processing_time": 3.518,
    "nrows": 3,
    "nsuccess": 3,
    "nfail": 0,
    "nskip": 0
  },
  "source": {
    "name": "/Users/dol/codes/data-prep-kit/transforms/language/pdf2parquet/test-data/input",
    "type": "path"
  },
  "target": {
    "name": "/Users/dol/codes/data-prep-kit/transforms/language/pdf2parquet/output",
    "type": "path"
  }
}