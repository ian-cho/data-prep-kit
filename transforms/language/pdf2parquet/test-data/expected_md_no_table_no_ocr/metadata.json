{
  "pipeline": "pipeline_id",
  "job details": {
    "job category": "preprocessing",
    "job name": "pdf2parquet",
    "job type": "pure python",
    "job id": "job_id",
    "start_time": "2025-02-10 14:44:09",
    "end_time": "2025-02-10 14:44:11",
    "status": "success"
  },
  "code": {
    "github": "github",
    "commit_hash": "12345",
    "path": "path"
  },
  "job_input_params": {
    "batch_size": -1,
    "artifacts_path": null,
    "contents_type": "text/markdown",
    "do_table_structure": false,
    "do_ocr": false,
    "ocr_engine": "easyocr",
    "bitmap_area_threshold": 0.05,
    "pdf_backend": "dlparse_v2",
    "double_precision": 0,
    "checkpointing": false,
    "max_files": -1,
    "random_samples": -1,
    "files_to_use": [
      ".pdf",
      ".docx",
      ".pptx",
      ".zip"
    ],
    "num_processors": 0
  },
  "execution_stats": {
    "cpus": 28.8,
    "gpus": 0,
    "memory": 22.7,
    "object_store": 0,
    "execution time, min": 0.038
  },
  "job_output_stats": {
    "source_files": 2,
    "source_size": 605137,
    "result_files": 2,
    "result_size": 29781,
    "processing_time": 1.506,
    "nrows": 3,
    "nsuccess": 3,
    "nfail": 0,
    "nskip": 0
  },
  "source": {
    "name": "/Users/dol/codes/data-prep-kit/transforms/language/pdf2parquet/test-data/input",
    "type": "path"
  },
  "target": {
    "name": "/Users/dol/codes/data-prep-kit/transforms/language/pdf2parquet/output",
    "type": "path"
  }
}