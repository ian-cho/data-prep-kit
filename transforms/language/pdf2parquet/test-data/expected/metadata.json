{
  "pipeline": "pipeline_id",
  "job details": {
    "job category": "preprocessing",
    "job name": "pdf2parquet",
    "job type": "pure python",
    "job id": "job_id",
    "start_time": "2025-02-10 14:18:13",
    "end_time": "2025-02-10 14:18:21",
    "status": "success"
  },
  "code": null,
  "job_input_params": {
    "batch_size": -1,
    "artifacts_path": null,
    "contents_type": "text/markdown",
    "do_table_structure": true,
    "do_ocr": true,
    "ocr_engine": "easyocr",
    "bitmap_area_threshold": 0.05,
    "pdf_backend": "dlparse_v2",
    "double_precision": 8,
    "checkpointing": false,
    "max_files": -1,
    "random_samples": -1,
    "files_to_use": [
      ".pdf",
      ".zip"
    ],
    "num_processors": 0
  },
  "execution_stats": {
    "cpus": 23.6,
    "gpus": 0,
    "memory": 29.99,
    "object_store": 0,
    "execution time, min": 0.127
  },
  "job_output_stats": {
    "source_files": 2,
    "source_size": 605137,
    "result_files": 2,
    "result_size": 32765,
    "processing_time": 3.93,
    "nrows": 3,
    "nsuccess": 3,
    "nfail": 0,
    "nskip": 0
  },
  "source": {
    "name": "/Users/dol/codes/data-prep-kit/transforms/language/pdf2parquet/test-data/input",
    "type": "path"
  },
  "target": {
    "name": "/Users/dol/codes/data-prep-kit/transforms/language/pdf2parquet/output",
    "type": "path"
  }
}